src/osdc/Objecter.cc

```diff
@@ -708,10 +708,9 @@ void Objecter::_send_linger_ping(LingerOp *info)
   o->target = info->target;
   o->should_resend = false;
   _send_op_account(o);
-  MOSDOp *m = _prepare_osd_op(o);
```

Comment:
for the posterity, it might not look like a refactor as it claims because the message's tid is set using o->tid before "o->tid" is updated. but please note, `Objecter::_send_op()` always `m->set_tid(op->tid)` before sending the message.

Reply:
@runsisi since you are at this file, could you help remove the line of https://github.com/ceph/ceph/blob/master/src/osdc/Objecter.cc#L3309 ? as `_prepare_osd_op()` should take care of this already.

Reply:
yeah, removed it since `_prepare_osd_op()` has set the updated `m->tid`.
thanks~